build.gradle

```diff
@@ -194,6 +194,9 @@ dependencies {
     compile 'io.springfox:springfox-swagger-ui:2.10.0'
     compile 'io.springfox:springfox-swagger2:2.9.2'
 
+    compile 'org.elasticsearch.client:elasticsearch-rest-high-level-client:7.10.0'
```

Comment:
We're not consistent about this but `implementation` is now the preferred way to declare a dependency. Gradle 7.0 doesn't support `compile`, so it'd be good to avoid it for new code.